nowcast.wsgi

```diff
@@ -72,7 +71,7 @@ def get_prob_dict(out_dir, lat, lon):
     timelist = list(map(lambda x: dt.datetime.strptime(x, timeformat), [i for i in data]))
 
     datetime_now = dt.datetime.utcnow()
-    timelist = list(filter(lambda x: datetime_now <= x and x <= datetime_now + dt.timedelta(hours=2), timelist))
+    timelist = list(filter(lambda x: datetime_now >= x and x <= datetime_now + dt.timedelta(hours=2), timelist))
```

Comment:
this was done to ensure rest api does not return data in the past, but I agree for your dataset it works only this way

Reply:
Yeah, how should we handle it then? Is it going to be used for realtime data or data from past, as in my case?

Reply:
Need additional parameter for 'asoftime' in rest api, by default equal to current datetime, to be able to pass it and test data in the past. This will also assist in resolving production issues, as currently there is no way to test behaviour in the past